src/edu/mills/cs180a/RomanNumeral.java

```diff
@@ -1,6 +1,5 @@
 package edu.mills.cs180a;
-
-import java.util.Map;
+import java.util.*;
```

Comment:
Put a blank line after `package`. (I'm surprised the auto-format didn't fix this.)